package.json

```diff
@@ -31,10 +31,15 @@
     "copy:readium-css": "cp -r ./node_modules/readium-css/css/dist examples/streamed/readers/reader-JBKS/readium-css",
     "copy:r2-glue-js": "cp ./node_modules/r2-glue-js/dist/ReadiumGlue.js examples/streamed/readers/reader-JBKS/r2-glue-js.js",
     "copy:r2-navigator": "mkdir -p examples/streamed/readers/reader-JBKS/@readium && cp ./node_modules/@readium/navigator-web/dist/readium-navigator-web.umd.js examples/streamed/readers/reader-JBKS/@readium/navigator-web.js",
-    "copy:streamed": "cpx 'viewer/*.{html,css,js}' examples/streamed/readers/reader-JBKS"
+    "copy:streamed": "cpx 'viewer/*.{html,css,js}' examples/streamed/readers/reader-JBKS",
+    "eb-start": "npm run prepare && npm run examples && npm run streamed"
```

Comment:
Is this meant to be temporary? In a production env, you wouldn't want examples to run.

Reply:
So from what I understand of the way webpub-viewer worked, you had to build the examples to be able to run streamed.  I'll look into why this is, I agree it's not necessary to run all the examples.